app/models/bulkrax/entry.rb

```diff
@@ -85,20 +85,19 @@ def status_info(e = nil)
         self.last_error = nil
         self.last_error_at = nil
         self.last_exception = nil
-        self.last_succeeded_at = Time.now
+        self.last_succeeded_at = Time.current
       else
         self.last_error =  { error_class: e.class.to_s, error_message: e.message, error_trace: e.backtrace }
-        self.last_error_at = Time.now
+        self.last_error_at = Time.current
```

Comment:
Will these changes break anything? 
Rubocop complained about `Time.now` and provided a list of acceptable alternatives (one I considered was `Time.now.utc`). If there is a preference here, let me know.

Reply:
Time.current seems fine.